snare/cloner.py

```diff
@@ -14,6 +14,8 @@
 
 class Cloner(object):
     def __init__(self, root, max_depth, css_validate):
+        self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.DEBUG)
```

Comment:
This doesn't work (at least for me). Have you checked this?

I guess the problem might be in the parent logger, which has a level ERROR

Reply:
Umm. It works for me though.

Let me write a test for this.

Reply:
I have no idea why it doesn't work for me... Let me test on another env

Reply:
Alright :)

Reply:
Ok, now there is no print in console, but still, there are no cloned page logs in log

Reply:
I am not sure what's wrong. Are tests passing on your side?

Reply:
I run `clone --target example.com` and I want to see 2 file names in the log. And the log file is empty

Reply:
Tests are passing

Reply:
When you run cloner, do you have records in the file?

Reply:
Sorry, I was not able to look into it because of my exams.

Will try to look today :)

Reply:
If you now set the level in the logger creation, you don't need it here

Reply:
I tried removing it but logger only works when this and parent logger are both set to DEBUG.

Reply:
@afeena Can you have a look?

Reply:
@rjt-gupta only after 26th of September, until then I am not available. If someone can take a look before, would be nice :) @rnehra01 @glaslos